docs/source/starthere/intro.rst

```diff
@@ -8,42 +8,125 @@ Introduction
+
+Installation
+------------
+
+**Using NVIDIA PyTorch Container**
+
+To leverage all optimizations for LLM training, including 3D Model Parallel, fused kernels, FP8, and more, we recommend using the NVIDIA PyTorch container.
+
+.. code-block:: bash
+
+    docker pull nvcr.io/nvidia/pytorch:24.01-py3
+    docker run --gpus all -it nvcr.io/nvidia/pytorch:24.01-py3
+
+Within the container, you can install NeMo and its dependencies as follows:
+
+NeMo Installation
+
+.. code-block:: bash
+
+    apt-get update && apt-get install -y libsndfile1 ffmpeg
+    pip install Cython
+    pip install nemo_toolkit['all']
+
+Transformer Engine Installation
+
+This step involves cloning the Transformer Engine repository, checking out a specific commit, and installing it with specific flags.
+
+.. code-block:: bash
+
+    git clone https://github.com/NVIDIA/TransformerEngine.git && \
+    cd TransformerEngine && \
+    git fetch origin 8c9abbb80dba196f086b8b602a7cf1bce0040a6a && \
+    git checkout FETCH_HEAD && \
+    git submodule init && git submodule update && \
+    NVTE_FRAMEWORK=pytorch NVTE_WITH_USERBUFFERS=1 MPI_HOME=/usr/local/mpi pip install .
+
+Apex Installation
+
+This step includes a bug fix for Apex in the PyTorch 23.11 container.
+
+.. code-block:: bash
+
+    git clone https://github.com/NVIDIA/apex.git && \
+    cd apex && \
+    git checkout c07a4cf67102b9cd3f97d1ba36690f985bae4227 && \
+    cp -R apex /usr/local/lib/python3.10/dist-packages
+
+PyTorch Lightning Installation
+
+This involves installing a bug-fixed version of PyTorch Lightning from a specific branch.
+
+.. code-block:: bash
+
+    git clone -b bug_fix https://github.com/athitten/pytorch-lightning.git && \
+    cd pytorch-lightning && \
+    PACKAGE_NAME=pytorch pip install -e .
+
+Megatron Core Installation
+
+This section details the steps to clone and install the Megatron Core.
+
+.. code-block:: bash
+
+    git clone https://github.com/NVIDIA/Megatron-LM.git && \
+    cd Megatron-LM && \
+    git checkout a5415fcfacef2a37416259bd38b7c4b673583675 && \
+    pip install .
+
+AMMO Installation
+
+Finally, this step involves installing the AMMO package.
+
+.. code-block:: bash
+
+    pip install nvidia-ammo~=0.7.0 --extra-index-url https://pypi.nvidia.com --no-cache-dir
+
 
-#. You have Python version 3.10 or above.
+.. code-block:: bash
 
-#. You have Pytorch version 1.13.1 or 2.0+.
+    apt-get update && apt-get install -y libsndfile1 ffmpeg
+    pip install Cython
+    pip install nemo_toolkit['all']
+
+**Conda Installation**
+
+We recommend installing NeMo in a fresh Conda environment for those not using the NVIDIA PyTorch container.
+
+.. code-block:: bash
 
-#. You have access to an NVIDIA GPU, if you intend to do model training.
+    conda create --name nemo python==3.10.12
+    conda activate nemo
 
-.. _quick_start_guide:
+Follow the PyTorch `configurator <https://pytorch.org/get-started/locally/>`_ to install PyTorch.
 
 Quick Start Guide
```

Comment:
I think it would be better if this "quick start" stuff was above installation stuff, would be more attention-grabbing.
Although maybe this quickstart stuff doesn't belong in dev-docs and belongs in user guide instead?